drivers/flash/Kconfig.nordic_qspi_nor

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig NORDIC_QSPI_NOR
+	bool "QSPI NOR Flash"
+	select FLASH_HAS_DRIVER_ENABLED
+	select NRFX_QSPI
+	depends on HAS_HW_NRF_QSPI
+	help
+	  Enable support for nrfx QSPI driver with EasyDMA.
+
+if NORDIC_QSPI_NOR
+
+config NORDIC_QSPI_NOR_QE_BIT
+	int "Quad Enable bit number in Status Register"
+	default 6
+
+config NORDIC_QSPI_NOR_INIT_PRIORITY
+	int
```

Comment:
As this option has no prompt, users will not be able to modify it in menuconfig or in an application's `prj.conf`. Was that the intent? I see that in `Kconfig.nor` the situation is the same, but I'm actually not sure if that was intended there either.

Reply:
I did it intentionally to provide consistency with `Kconfig.nor`. It could be changed, but I think both changes (in `Kconfig.nordic_qspi_nor` and `Kconfig.nor`) schould be done in one commit in another PR (since this one does not touch `spi_nor`).

Reply:
I agree with that.  I doubt it was intentional in either case.  I can change it in both later, or we could make progress on https://github.com/zephyrproject-rtos/zephyr/issues/20348#issuecomment-564108715 which could lead to automatic management of initialization order based on dependencies.